docs/developers/contributing.md

```diff
@@ -29,8 +29,7 @@ Bugfixes should include a regression test which prevents that bug from being re-
 
 ## Update Documentation
 
-Documentation for the project is stored in the [docs](..) directory. If your feature requires docs updates,
-those updates must be submitted in the same Pull Request as the code changes.
+Documentation for the project is stored in the [prebid-server directory](https://github.com/prebid/prebid.github.io/tree/master/prebid-server) in the prebid.js repo. If your feature requires docs updates, those updates must be submitted in the same Pull Request as the code changes.
```

Comment:
These raw docs are mirrored on github.org? If so we probably want to provide both links, the above for editing, and github.org for viewing.